docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -51,11 +51,11 @@ The **why** is covered in the [release blog post](/blog/mui-core-v5/).
 
 ## `ThemeProvider` setup
 
-Make sure that `ThemeProvider` is defined at the root of your application (even if you are using the **default theme**) and **NO** `useStyles` is called before `<ThemeProvider>`.
-This is because we are going to use `@mui/styles` temporarily, which requires `ThemeProvider`.
+Before upgrading to v5, please make sure that `ThemeProvider` is defined at the root of your application (even if you are using the **default theme**) and **NO** `useStyles` is called before `<ThemeProvider>`.
+This is because we are going to use `@mui/styles` **temporarily** (JSS style-engine), which requires `ThemeProvider`.
 
 ```js
-import { ThemeProvider, createMuiTheme, makeStyles } from '@mui/material/styles';
+import { ThemeProvider, createMuiTheme, makeStyles } from '@material-ui/core/styles';
```

Comment:
This is the point identified in https://github.com/mui-org/material-ui/issues/28496. At this point, the developer is still using v4 so it should be `@material-ui/core/styles` instead of the new package.